example-data/turtle/sio-model/diseaseHistoryAndDiagnosis.ttl

```diff
@@ -1,19 +1,29 @@
 @prefix : <http://purl.org/ejp-rd/cde/v020/example-rdf/> .
-@prefix obo: <http://purl.obolibrary.org/obo/> .
 @prefix sio: <http://semanticscience.org/resource/> .
-@prefix dct: <http://purl.org/dc/terms/> .
+@prefix obo: <http://purl.obolibrary.org/obo/> .
+@prefix edam: <http://edamontology.org/> .
 @prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
 
 
-:person1 a obo:NCBITaxon_9606;
-  sio:SIO_000228 :role3 .
+:person_ a sio:SIO_000498 ;
+    sio:SIO_000228 :role_ ;
+    sio:SIO_000217 :quality_1_, :quality_2_ .
+  
+:identifier_ a edam:data_0842 ;
```

Comment:
I think we can remove `:identifier_` node from this turtle file. [Reason] same as previous comments

Reply:
I can do that. The identifier nodes are there because they were also in the previous TTL examples I updated ;) . But I agree that could be removed.